src/libstore/filetransfer.cc

```diff
@@ -364,6 +381,8 @@ struct curlFileTransfer : public FileTransfer
                 curl_easy_setopt(req, CURLOPT_READFUNCTION, readCallbackWrapper);
                 curl_easy_setopt(req, CURLOPT_READDATA, this);
                 curl_easy_setopt(req, CURLOPT_INFILESIZE_LARGE, (curl_off_t) request.data->length());
+                curl_easy_setopt(req, CURLOPT_SEEKFUNCTION, seekCallbackWrapper);
```

Comment:
Is there a way we can test this in our nix test suite? Alternative: Is there a way I can manually test this?

Reply:
Can try it with this: https://gist.github.com/tomberek/60cd89c246d436dbb5ad5fe1a0805dd0